app/inpututils.cpp

```diff
@@ -286,8 +286,13 @@ bool InputUtils::cpDir( const QString &srcPath, const QString &dstPath )
     {
       if ( !QFile::copy( srcItemPath, dstItemPath ) )
       {
-        qDebug() << "Cannot copy file " << srcItemPath << " > " << dstItemPath;
-        return false;
+        QFile::remove( dstItemPath );
```

Comment:
Would be good to have error handling here as well (and the same for other error messages in this function)